cilium/CFP-30984-dns-proxy-ha-v2.md

```diff
@@ -0,0 +1,109 @@
+# CFP-30984: toFQDN DNS proxy HA
+
+**SIG: SIG-POLICY**
+
+**Begin Design Discussion:** 2024-08-19
+
+**Cilium Release:** 1.17
+
+**Authors:** Hemanth Malla <hemanth.malla@datadoghq.com>, Vipul Singh <singhvipul@microsoft.com>
+
```

Comment:
Administrative point:

If we think we agree on the basics here, but would like more of the design to be fleshed out:

```suggestion

Status: Provisional

```
(If provisional, please highlight anything that still needs to be expanded with the text "UNRESOLVED: _description of unresolved design point_").

Or alternatively if we think the design is satisfactorily complete:


```suggestion

Status: Implementable

```

See status in the readme for more details. https://github.com/cilium/design-cfps?tab=readme-ov-file#status

Reply:
This was the point I made at the end of the discussion with @cilium/sig-policy today. Whatever we agree upon, we should be able to merge in. If there are points left to iterate on, we can highlight those points. If we've resolved the points of difference we can mark it implementable. Goal is to make iteration easier so we don't have to overthink an entire design that is subject to change during implementation, but rather we can more incrementally change the design in-tree as we discuss, debate, implement and learn.

Reply:
I'll add a status field once we close out the open discussion items.

Reply:
Also looks like we haven't documented `Provisional ` in the readme yet ?

Reply:
Also looks like we haven't documented `Provisional ` in the readme yet ?

Reply:
Ah right, provisional is just under discussion in https://github.com/cilium/design-cfps/issues/48 but not documented yet.